docs/building-spark.md

```diff
@@ -219,7 +219,7 @@ The run-tests script also can be limited to a specific Python version or a speci
 ## Running R Tests
 
 To run the SparkR tests you will need to install the R package `testthat`
-(run `install.packages(testthat)` from R shell).  You can run just the SparkR tests using
+(run `install.packages("testthat")` from R shell).  You can run just the SparkR tests using
```

Comment:
Mind updating this contents to be consistent with https://github.com/apache/spark/blob/7e0cd1d9b168286386f15e9b55988733476ae2bb/R/README.md#examples-unit-tests if it sounds making sense to you?

This also looks including specifying the mirror from where to download the package.